pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -187,12 +188,31 @@ func addFileWritePermissionToUser(filename string) error {
 }
 
 func stopCRCHyperkitProcess() error {
-	path, err := exec.LookPath("pkill")
+	pgrepPath, err := exec.LookPath("pgrep")
+	if err != nil {
+		logging.Debugf("Could not find 'pgrep'. %v", err)
```

Comment:
Why we want to have `Debugf` if returning the same string as error .

Reply:
I did not check if this error was printed or not, it's just a copy and paste of the 'pkill' codepath which already has it.

Reply:
`doCleanUpPreflightChecks` prints the error, so I can remove these.

Reply:
yes, let's remove duplicate debug if it have same strings.